rcm_nexus/command.py

```diff
@@ -38,6 +39,29 @@ def init():
     https://mojo.redhat.com/docs/DOC-1199638
     """ % conf_path)
 
+
+@click.command()
+def list_of_commands():
+    """Commands available:
+
+     - nexus - this command - list of available actions
+
+     - nexus-add-npm-product - adds a new npm product
+
+     - nexus-add-product - adds a new Maven product (and staging profile)
```

Comment:
TODO: We should deprecate generic commands like this in favor of commands that reference specific package types

Reply:
Good idea. I will change it to nexus-add-java-product

Reply:
I added one commit, that changes this (+ the comment on the next comment (if npm_archive_type != npm.NpmArchiveType.NOT_NPM:)